src/tabs/tabs.scss

```diff
@@ -4,100 +4,151 @@
 
 @mixin iui-tabs {
   @include iui-reset;
-
-  @include iui-font-family;
+  position: relative;
   display: flex;
+  align-items: center;
   list-style: none;
-  padding: 0;
-  margin: round($iui-baseline/2) 0 0 0;
   user-select: none;
 
-  > li {
-    display: inline-block;
+  .iui-tab {
+    @include iui-reset;
+    @include iui-font-family;
+    @include iui-focus;
```

Comment:
Tab focus is working now that I'm using `::after { content` to create the active tab stripe.  However, we do have some clipping, also present in other parts of iTwinUI (header, grouped buttons, etc).  This could be fixed in #166.
![Screen Shot 2021-06-22 at 6 28 22 PM](https://user-images.githubusercontent.com/849817/123007731-a360ae00-d387-11eb-8032-1aa48e974c8c.png)